### DISCUSSION TITLE
How to enable src/dst IP and port in pcap

### DISCUSSION THREAD

**Discussion by SadeghKrmi** [srsran/srsRAN_Project#129] [General Help]
How is it possible to have port and IP addresses in the pcap traces? for example in the following snapshot which is gnb_ngap.pcap

![image](https://github.com/srsran/srsRAN_Project/assets/5988663/1b98b156-8a06-4a17-8c2a-8d3e2106ee29)

**Comment by andrepuschmann** [comment]
We don't fill those fields when writing the PCAP as we capture this on NGAP level the information is actually already gone. But if you need that you could also simply fire up Wireshark and capture on the actual interface.

**Reply by SadeghKrmi** [reply]
@andrepuschmann what if capture is being done on the whole interface, how to decode f1ap and e1ap etc.?

**Reply by SadeghKrmi** [reply]
@andrepuschmann I am attaching the capture, please help if you know how to decode f1ap in the pcap.
[capture_network.zip](https://github.com/srsran/srsRAN_Project/files/12085600/capture_network.zip)

**Reply by wmwwimor** [reply]
Could you tell me how to grasp data  from all of interfaces by wireshark in real-time?Thx!!

**Reply by andrepuschmann** [reply]
@SadeghKrmi Right now you can't capture F1 or E1 on the network interface because they are not send over the network.
The trace above is a NGAP PCAP. You can look at it using the internal capture (e.g. the file in `/tmp/gnb_ngap.pcap`) or using Wireshark on the interface.
